src/test/java/com/google/googleidentity/oauth2/endpoint/TokenEndpointTest.java

```diff
@@ -110,13 +103,8 @@ public void init() {
     clientSession = new ClientSession();
     clientSession.setClient(CLIENT);
     System.setProperty("AUTH_CODE_LENGTH", "10");
-    Map<GrantType, RequestHandler> map = new HashMap<>();
-    map.put(
-        GrantType.AUTHORIZATION_CODE,
-        new AuthorizationCodeRequestHandler(
-            new AuthorizationCodeService(new InMemoryCodeStore()),
-            new InMemoryOAuth2TokenService()));
-    tokenEndpoint = new TokenEndpoint(clientDetailsService, new MultipleRequestHandler(map));
+    RequestHandler requestHandler = mock(RequestHandler.class);
```

Comment:
Same here, I think you need to set the expected request and response pair in order to pass the test cases.

Reply:
The handler's responses are tested in other tests.